docs/how-to/make-publishable-ioc.rst

```diff
@@ -0,0 +1,35 @@
+Create a Publishable IOC
+========================
+
+As seen in :doc:`../tutorials/creating-an-ioc`, a single Python script can be an IOC.
```

Comment:
```suggestion
As seen in `creating-an-ioc`, a single Python script can be an IOC.
```
Does this work?

Reply:
I can't remove the "../tutorials/" part without encountering this error:
`WARNING: 'any' reference target not found: creating-an-ioc`

From my understanding of the [:any: ](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-any)and the [:doc: ](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-doc) roles, it looks like the full path is always required but the :doc: can be inferred from a lack of any role prefix. 

I'll go through and delete all the role tags I can.

Reply:
Resolved in commit a873d0e